examples/AzureML-REST-API/submit-custom-image.md

```diff
@@ -78,12 +78,49 @@ resp = requests.post(start_run, files=run_files, headers=header)
 print("response text: ", json.loads(resp.text))
 ```
 
+## Tag the run
+
+Tagging runs lets you later retrieve runs by more meaningful identifiers than the run ID.
+
+```python
+run_id = resp.json()["runId"]
+
+run_uri = hosturl + historybase + resourcebase + "experiments/{}/runs/{}".format(experiment_name,run_id)
+payload = {
+   "tags": {
+      "customimage": "specialrun"
+   }
+}
+
+# If you forget the content type header, you'll get an HTTP 415
+header.update({
+    "content-type": "application/json"
+})
+resp = requests.patch(run_uri, json=payload, headers=header)
+print(resp.json()["tags"])
+```
+
+## Retrieve a run by tag
+
+Later, you can fetch the run by querying the experiment
+
+```python
+query_runs = hosturl + historybase + resourcebase + "experiments/{}/runs:query".format(experiment_name)
+payload = {
+   "filter": "Tags/customimage eq specialrun",
+    "top": 1
+}
+
+resp = requests.post(query_runs, json=payload, headers=header)
+print(resp.json()["value"][0])
+```
+
 ## Monitor status of run
 
 You can track the status of the run by polling the run history service.
 
 ```python
-run_id = json.loads(resp.text)["runId"]
+run_id = resp.json()["value"][0]["runId"]
```

Comment:
Would be good to show an example response

Reply:
What would you like to see here? The result of this is a RunDto which they would get from the previous walkthroughs, and the upperlevel response here is a paginatedresult, which is a list of RunDto effectively. Happy to add examples of either if you think they're useful

Reply:
I was thinking just show an an example of a status returned